students/tfbanks/Lesson09/Mailroom_pt5.py

```diff
@@ -0,0 +1,124 @@
+# Mailroom_pt5py Exercise by tfbanks
+
+# !/usr/bin/env python3
+
+import os
+# Donor Database containing names and donations
+
+
+
+class Donor:
+    def __init__(self, donations=None):
+        self.donations = donations if donations else []
```

Comment:
your Donor class should have more attributes and methods pertaining to donor object like:
* donor name
* avg donations
* total donations
* sum donations
* thank you letter text